stable/gcloud-endpoints/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+        - name: endpoints
+          image: "{{ .Values.image }}"
+          imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+          args: [
+{{- if (.Values.httpPort) }}
+            "--http_port", {{ default "" .Values.httpPort | quote }},
+{{- end }}
+{{- if (.Values.http2Port) }}
+            "--http2_port", {{ default "" .Values.http2Port | quote }},
+{{- end }}
+{{- if (.Values.sslPort) }}
+            "--ssl_port", {{ default "" .Values.sslPort | quote }},
+{{- end }}
+{{- if (.Values.backend) }}
+            "--backend", {{ default "" .Values.backend | quote }},
+{{- end }}
+{{- if (.Values.service) }}
+            "--service", {{ default "" .Values.service | quote }},
+{{- end }}
+{{- if (.Values.version) }}
+            "--version", {{ default "" .Values.version | quote }},
+{{- end }}
+{{- if (.Values.serviceAccountKey) }}
+            "--service_account_key", {{ default "" .Values.serviceAccountKey | quote }},
+{{- end }}
+{{- if (.Values.nginxConfig) }}
+            "--nginx_config", {{ default "" .Values.nginxConfig | quote }},
+{{- end }}
+{{- if (.Values.statusPort) }}
+            "--status_port", {{ default "" .Values.statusPort | quote }},
+{{- end }}
+          ]
+          ports:
+{{- if (.Values.httpPort) }}
```

Comment:
Can you implement `.Values.httpPort` as an array? Then you could support any number of ports:
```
{{ $port, $name := range .Values.httpPort }}
  - containerPort: {{ $port }}
     name: {{ $name }}

```